### PR TITLE
feat: rework docs structure and improve discoverability

### PR DESCRIPTION
**Pull request by noobinthisgame**
## 📝 Description

This PR reworks the structure of the docs to improve the discoverability of content and improve the UX. Also this rework aims to make it easier for new chakra user to get into chakra.

## 💣 Is this a breaking change (Yes/No):

Yes, if people might have used links somewhere to reference pages in the docs, those pages might have moved

### CODE REVIEW

**Comment by noobinthisgame** [comment]
@nikolovlazar I'd be glad to hear your thoughts about this first iteration of the new structure. The overview for API and Components pages would be something I'd do in a following PR.

**Comment by nikolovlazar** [comment]
Looking good, @noobinthisgame! Love it! I suggest we wait for the "Overview" PR before we merge this one.

**Comment by noobinthisgame** [comment]
Yes agreed :) I'll start working on it asap. I might need help for the images.

**Comment by nikolovlazar** [comment]
@noobinthisgame i can help you out with that.

File: contentlayer.config.ts
Comment (nikolovlazar) [inline contentlayer.config.ts:92]: You might've forgotten a `console.log` here.

File: layouts/mdx.tsx
Comment (nikolovlazar) [inline layouts/mdx.tsx:12]: This looks like an old typo, `sidebat`.

File: pages/docs/[[...slug]].tsx
Comment (nikolovlazar) [inline pages/docs/[[...slug]].tsx:3]: What happened with the import alias here?

File: src/components/mobile-nav.tsx
Comment (nikolovlazar) [inline src/components/mobile-nav.tsx:31]: We should move this import to the top to separate it from the local imports.

File: src/components/mobile-nav.tsx
Comment (nikolovlazar) [inline src/components/mobile-nav.tsx:76]: I can't see where we use the `id` here.

File: pages/guides/installation.mdx
Comment (nikolovlazar) [inline pages/guides/installation.mdx:23]: We're showing a blockquote that says that you need to set "this" up in a certain way for CRA. But, since we moved the instructions out of this file "this" refers to nothing now.

Comment (nikolovlazar) [reply pages/guides/installation.mdx:23]: Also, I think we should merge the `/guides/integrations/with-cra` and `/guides/getting-started/cra-guide` pages.

File: src/components/mobile-nav.tsx
Comment (noobinthisgame) [reply src/components/mobile-nav.tsx:31]: We could add a eslint rule for this in a new PR, to make sure it is marked in the future.

File: pages/guides/installation.mdx
Comment (noobinthisgame) [reply pages/guides/installation.mdx:23]: Right good Idea!

Comment (noobinthisgame) [reply pages/guides/installation.mdx:23]: I'll put both under getting-started, what do you think ?

**Comment by noobinthisgame** [comment]
@nikolovlazar This is ready for another round of reviews :)

File: pages/guides/first-steps.mdx
Comment (nikolovlazar) [inline pages/guides/first-steps.mdx:2]: Let's change this to "First steps"

File: layouts/mdx.tsx
Comment (nikolovlazar) [inline layouts/mdx.tsx:9]: Let's separate this third-party import from the local ones.

**Review by nikolovlazar — CHANGES_REQUESTED** [review]
Veeery small fixes, and then we merge 😅

**Comment by noobinthisgame** [comment]
@nikolovlazar Changes are applies ;)

**Pull request merged**
